test/pgtest/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/testtablespace)
 # Tests to ignore
 set(PG_IGNORE_TESTS
   advisory_lock
+  event_trigger
   foreign_data
```

Comment:
Accidental addition? Does not look related.

Reply:
This is actually related since latest PG versions print a list of all defined event triggers in the event_trigger test making the output different when timescaledb is installed since we create our own event triggers.

https://github.com/postgres/postgres/commit/f3c4537876843bea9a9d35d9643d347750961848

Reply:
Ah, ok.